uMatriks/Login.qml

```diff
@@ -136,6 +135,18 @@ Page {
             placeholderText: qsTr("Password")
         }
 
+        Label {
+            id: homeserverLabel
+            text:i18n.tr("Home server URL:")
+        }
+
+        TextField {
+            id: homeserverField
+            inputMethodHints: Qt.ImhNoAutoUppercase
+            width: parent.width
+            placeholderText: qsTr("https://matrix.org")
```

Comment:
Out of curiosity, what is the difference between `i18n.tr` and `qsTr`? In any case, making a URL translatable looks very suspicious.

Reply:
I overlooked that in my copy/paste from the other field shenanigans.  I was going for consistency in the existing code and overlooked the fact that it's a URL.  I'll fix that.